src/MetalPerformanceShaders/MPSImageLanczosScale.cs

```diff
@@ -1,19 +1,19 @@
+#if XAMCORE_2_0 || !MONOMAC
 // Copyright 2015 Xamarin Inc. All rights reserved.
 
-#if IOS
-
 using System;
 using System.Runtime.InteropServices;
 using XamCore.Foundation;
 using XamCore.Metal;
 using XamCore.ObjCRuntime;
 
 namespace XamCore.MetalPerformanceShaders {
-
+	[Mac (10, 13)]
```

Comment:
it's a `partial` type so it's not needed everywhere - and better placed in the binding (not manual) code along with other availability attributes.